internal/configtxgen/genesisconfig/config.go

```diff
@@ -217,21 +231,48 @@ func LoadTopLevel(configPaths ...string) *TopLevel {
 
 	err := config.ReadInConfig()
 	if err != nil {
-		logger.Panic("Error reading configuration: ", err)
+		logger.Panicf("Error reading configuration: %s", err)
 	}
 	logger.Debugf("Using config file: %s", config.ConfigFileUsed())
 
-	var uconf TopLevel
-	err = viperutil.EnhancedExactUnmarshal(config, &uconf)
+	uconf, err := cfgCache.loadConfigCache(config, config.ConfigFileUsed())
```

Comment:
`cfgCache.loadConfigCache(config, config.ConfigFileUsed())` - seems to stutter. Why isn't it just `load` ?